package.json

```diff
@@ -14,7 +14,7 @@
 	"exports": "./index.js",
 	"types": "./index.d.ts",
 	"engines": {
-		"node": ">=14.16"
+		"node": ">=18"
```

Comment:
Breaking change - but looking at other packages I think this would be an acceptable one?

Reply:
Yup